compiler/rustc_feature/src/unstable.rs

```diff
@@ -213,6 +213,8 @@ declare_features! (
     (internal, negative_bounds, "1.71.0", None),
     /// Allows using `#[omit_gdb_pretty_printer_section]`.
     (internal, omit_gdb_pretty_printer_section, "1.5.0", None),
+    /// Highlights the error message of pannic calls.
+    (unstable, panic_color_errors,"CURRENT_RUSTC_VERSION", None),
```

Comment:
This is only necessary for compiler features. Library features like this PR should not get an entry in the compiler.

Reply:
Okay, so its only that part or there's more we should delete?

Actually, we found this [guide](https://rustc-dev-guide.rust-lang.org/implementing_new_features.html) somewhat confusing regarding library features. The link it provides for the library features guide leads only to stability attributes. As a result, we followed the guide for compiler features instead.

Reply:
It should really link to https://std-dev-guide.rust-lang.org/development/feature-lifecycle.html